lib/phx_tools_web/live/phx_tools_live/index.ex

```diff
@@ -31,28 +29,12 @@ defmodule PhxToolsWeb.PhxToolsLive.Index do
     end
   end
 
-  defp apply_action(socket, action) do
-    {:noreply,
-     socket
-     |> assign(seo_attributes: %{url: Endpoint.url() <> "/#{action}"})
-     |> assign(:source_code_url, source_code_url_from_os_or_live_action(action))}
-  end
-
-  defp installation_command(live_action) do
-    "$SHELL -c \"$(curl -fsSL #{get_script_url(live_action)})\""
-  end
-
-  defp source_code_url_from_os_or_live_action(os_or_live_action) do
-    ~S(https://github.com/optimumBA/phx.tools/blob/main/priv/static/) <>
-      get_script_filename(os_or_live_action)
-  end
-
-  defp get_script_url(live_action_or_os) do
-    "#{Endpoint.url()}/#{get_script_filename(live_action_or_os)}"
+  defp apply_action(socket, _action) do
```

Comment:
We don't need this anymore, nor the `live_action` concept as there's only one page in the whole website.

Reply:
We need the live action concept to determine whether to show the unsupported os message or not but we can move the implementation to the mount function

Reply:
It shouldn't be called `live_action` anymore.
`live_action` is a special name used by Phoenix LiveView.

Reply:
Okay, maybe we can use the name of the operating system instead.

Reply:
@almirsarajcic what do you think?

Reply:
Make the changes and I'll tell you.